packages/wonder-blocks-modal/index.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import ModalLauncher from "./components/modal-launcher.js";
```

Comment:
You could also rewrite each of these lines as:

`export {default as ModalLuauncher} from "./components/modal-launcher.js";`

Reply:
I'll do a separate PR to do that across all packages.